cone.js

```diff
@@ -118,4 +118,11 @@ module.exports = function(vectorfield, bounds) {
 	return geo;
 };
 
-module.exports.createConeMesh = require('./lib/conemesh');
+var shaders = require('./lib/shaders');
+module.exports.createMesh = require('./lib/conemesh');
```

Comment:
Can we call this file `create_mesh.js` instead of `lib/conemesh.js` if it's going to be used for streamtubes too?

Reply:
Good call. 
Planned to do in the next PR.